app/assets/javascripts/webauthn.js

```diff
@@ -46,6 +47,23 @@
     });
   };
 
+  var parseCreationOptionsFromJSON = function(json) {
```

Comment:
Is there a special reason why we want to use `var` here instead of `let`, or even better `const`? My understanding is that `var` has really bad semantics in JS (especially wrt scoping) and that modern code should switch over to using either `let` or `const`. Also, I see a usage of `const` further down in this file, so I suspect it is not a compatibility concern.

Reply:
good catch, i had just copied from elsewhere in the file. I updated everything to be `const` now.